docs/data-sources/cloud_backup_snapshot_export_bucket.md

```diff
@@ -35,7 +35,7 @@ In addition to all arguments above, the following attributes are exported:
 * `cloud_provider` - Name of the provider of the cloud service where Atlas can access the S3 bucket.
 * `role_id` - Unique identifier of the Azure Service Principal that Atlas can use to access the Azure Blob Storage Container.
 * `service_url` - URL that identifies the blob Endpoint of the Azure Blob Storage Account.
-* `tenant_id` - UUID that identifies the Azure Active Directory Tenant ID.
+* `tenant_id` - (Deprecated) This field is ignored; the tenantId of the Cloud Provider Access role (from roleId) is used. UUID that identifies the Azure Active Directory Tenant ID.
```

Comment:
can we reference the cloud provider access role attribute name here instead? `the tenantId of the Cloud Provider Access role (from roleId)` isn't very intuitive

Reply:
I'm folowing the [API](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Cloud-Backups/operation/createExportBucket) message when `cloudProvider=AZURE`.
However, `mongodbatlas_cloud_provider_access_authorization.azure.tenant_id` might be more intuitive. WDYT?

Reply:
yess that works!

Reply:
Updated in 2474fb9f1044fe5a84eb1bfe4f7c99682e51ab29